doc/source/user_guide/io.rst

```diff
@@ -2,12 +2,11 @@
 
 .. currentmodule:: pandas
 
-
-{{ header }}
-
 .. ipython:: python
    :suppress:
 
+   import pandas as pd
+   pd.options.display.max_rows = 15
    clipdf = pd.DataFrame({'A': [1, 2, 3], 'B': [4, 5, 6], 'C': ['p', 'q', 'r']},
                          index=['x', 'y', 'z'])
```

Comment:
Didn't realize this was hidden, thanks @jorisvandenbossche for pointing it out.

@TanyaaCJain this block is not shown to the user, can you move the `clipdf` data where it's first used, and the import to the first next block (assuming there are no more hidden blocks before.

Reply:
Made the changes in the commit 18a9b7c.